spec/presenters/summary/assessment_question_presenter_spec.rb

```diff
@@ -188,6 +164,79 @@
         include_examples 'format answered value'
       end
     end
+
+    context 'when question has a parent that is a question' do
+      let(:parent_value) { 'some_value' }
+      let(:parent) { instance_double(Assessments::Question, value: parent_value, is_question?: true) }
+
+      before do
+        expect(question).to receive(:belongs_to_group?).and_return(false)
+        allow(question).to receive(:parent).and_return(parent)
```

Comment:
The question itself receives a **parent** option as an argument of a new instance, there's no reason not to use that instead of stubbing the question. The less we stub, the better.